Source/Serilog.Exceptions/Destructurers/ExceptionDestructurer.cs

```diff
@@ -184,9 +172,35 @@ public virtual Type[] TargetTypes
                         typeof(System.UnauthorizedAccessException),
                         typeof(System.UriFormatException)
                     };
+
+#if NET45
+                foreach (var dangerousType in GetDangerousTypes())
+                {
+                    var type = Type.GetType(dangerousType);
+                    if(type != null)
+                    {
+                        targetTypes.Add(type);
+                    }
+                }
+#endif
+                return targetTypes.ToArray();
             }
         }
 
+        private string[] GetDangerousTypes()
```

Comment:
Thanks, I was struggling to find a good name bu wasn't thinking about Mono at all :smile: